app/src/main/java/org/oppia/android/app/maven/GenerateMavenDependenciesList.kt

```diff
@@ -0,0 +1,395 @@
+        )
+        throw Exception(
+          "The backup_license_links.json must " +
+            "contain an array with \"artifacts\" key."
+        )
+      }
+      val moshi = Moshi.Builder().addLast(KotlinJsonAdapterFactory()).build()
+      val adapter = moshi.adapter(BackupDependency::class.java)
+      val backupDependency = adapter.fromJson(backupJsonContent)
+      if (backupDependency == null) {
+        println(
+          "The backup_license_links.json file is empty. " +
+            "Please add the JSON structure to provide the BackupLicense Links."
+        )
+        throw Exception(
+          "The backup_license_links.json " +
+            "must contain an array with \"artifacts\" key."
+        )
+      }
+      backupLicenseLinksList = backupDependency.artifacts
+      if (backupLicenseLinksList.isEmpty()) {
+        println("The backup_license_links.json file does not contain any license links.")
+        return
+      }
+      backupLicenseLinksList.toSortedSet { license1, license2 ->
+        license1.artifactName.compareTo(
+          license2.artifactName
+        )
+      }
+      backupLicenseLinksList.forEach { license ->
+        backupLicenseDepsList.add(license.artifactName)
+      }
+      backupLicenseDepsList.sort()
+    }
+
+    fun parseArtifactName(artifactName: String): String {
+      var colonIndex = artifactName.length - 1
+      while (artifactName.isNotEmpty() && artifactName[colonIndex] != ':') {
+        colonIndex--
+      }
+      val artifactNameWithoutVersion = artifactName.substring(0, colonIndex)
+      val parsedArtifactNameBuilder = StringBuilder()
+      for (index in artifactNameWithoutVersion.indices) {
+        if (artifactNameWithoutVersion[index] == '.' || artifactNameWithoutVersion[index] == ':' ||
+          artifactNameWithoutVersion[index] == '-'
+        ) {
+          parsedArtifactNameBuilder.append(
+            '_'
+          )
+        } else {
+          parsedArtifactNameBuilder.append(artifactNameWithoutVersion[index])
+        }
+      }
+      return parsedArtifactNameBuilder.toString()
+    }
+
+    fun runBazelQueryCommand(command: String) {
+      val processBuilder = ProcessBuilder()
+      processBuilder.command("bash", "-c", command)
+      try {
+        val process = processBuilder.start()
+        val reader = BufferedReader(InputStreamReader(process.inputStream))
+        var line: String?
+        while (true) {
+          line = reader.readLine()
+          if (line == null) break
+          val endindex = line.toString().length
+          bazelQueryDepsNames.add(line.toString().substring(9, endindex))
+        }
+        bazelQueryDepsNames.sort()
+
+        val exitValue = process.waitFor()
+        if (exitValue != 0) {
+          System.err.println(
+            "There was some unexpected error while " +
+              "running the bazel Query command."
+          )
+          throw Exception("Unexpected error.")
```

Comment:
The command is returning non-zero exitValue even after changing the output_base.

Reply:
Try copying in BazelClient from #3374 and create any new query methods you need using the same approach as the other ones & see if that works.